src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AssertionAds.java

```diff
@@ -0,0 +1,19 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.ad;
+
+import com.wikia.webdriver.common.core.Assertion;
+import org.openqa.selenium.Dimension;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+public class AssertionAds {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1118)